bootstrap/src/main/java/io/airlift/bootstrap/Bootstrap.java

```diff
@@ -209,9 +210,15 @@ public Injector initialize()
 
         // initialize logging
         if (logging != null) {
```

Comment:
We should probably also check `initializeLogging` here, it seems like we don't want to configure logging if that's set.

Reply:
If `initializeLogging` is set, then `logging` will be too.

Reply:
More specifically, `logging` is only set if `initializeLogging` is set.

Reply:
Ah, yeah, it might read better if we just check the same boolean here...